### DISCUSSION TITLE
I am interested to get adopted with the technology but how to start?

### DISCUSSION THREAD

**Discussion by DeletedAccountMarch** [filecoin-project/community#368] [Developers]
Hello there,
This is me Arjun, Founder and CEO of dlupload.

Dlupload.com is a file sharing website where users upload their file and we store the file on our server but we are planning to make the file uploading and file sharing things decentralized. Users upload the file from my website and it should be stored on the decentralized blockchain or networks and for this server we are looking at your platform.

I am interested in bringing adoption to my company model, I hope you allow file sharing platforms to come to your platform? We do have daily 100 gb+ file uploads and it is getting even bigger.

We are interested to work with you, waiting to hear from you.
Please explain us if we can use this sevice.

Have a nice day!
Arjun Ghimire / Founder & CEO of dlupload

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
hey, "you" basically doesn't exist. filecoin is a community driven project. 

what are you up for? using filecoin as a storage solution? become a storage provider?
